diff --git a/node_modules/@skyware/labeler/dist/util/labels.js b/node_modules/@skyware/labeler/dist/util/labels.js
index b77f2f9..a09ff0e 100644
--- a/node_modules/@skyware/labeler/dist/util/labels.js
+++ b/node_modules/@skyware/labeler/dist/util/labels.js
@@ -3,7 +3,7 @@ import { k256Sign } from "./crypto.js";
 import { excludeNullish } from "./util.js";
 const LABEL_VERSION = 1;
 function formatLabelCbor(label) {
-    return excludeNullish({ ...label, ver: LABEL_VERSION, neg: !!label.neg });
+    return excludeNullish({ ...label, ver: LABEL_VERSION, neg: label.neg === true ? true : undefined });
 }
 export function formatLabel(label) {
     const sig = label.sig instanceof ArrayBuffer
@@ -14,7 +14,7 @@ export function formatLabel(label) {
     if (!sig || !("$bytes" in sig)) {
         throw new Error("Expected sig to be an object with base64 $bytes, got " + sig);
     }
-    return excludeNullish({ ...label, ver: LABEL_VERSION, neg: !!label.neg, sig });
+    return excludeNullish({ ...label, ver: LABEL_VERSION, neg: label.neg === true ? true : undefined, sig });
 }
 export function signLabel(label, signingKey) {
     const toSign = formatLabelCbor(label);
